### PR TITLE
[TOKENIZERS] Dynamic versions

### PR DESCRIPTION
**Pull request by mryzhov**
- Read version from version.py
- Added support of CI_BUILD_NUMBER for cpack packages

### CODE REVIEW

File: modules/custom_operations/CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,17 @@ if(NOT OpenVINODeveloperPackage_FOUND)
 endif()
 
 # set the project version using OpenVINO major, minor and patch versions
-set(CMAKE_PROJECT_VERSION_TWEAK 0)
-set(CMAKE_PROJECT_VERSION "${OpenVINO_VERSION}.${CMAKE_PROJECT_VERSION_TWEAK}")
+# tweak version is used to release extension independently from openvino package
+if (${SKBUILD})
```
Comment (ilya-lavrenov) [inline modules/custom_operations/CMakeLists.txt:19]:
```suggestion
if(DEFINED SKBUILD)
```

File: modules/custom_operations/CMakeLists.txt [hunk 2]
```diff
@@ -15,7 +15,17 @@ if(NOT OpenVINODeveloperPackage_FOUND)
...
+    set(CMAKE_PROJECT_VERSION "${SKBUILD_PROJECT_VERSION}")
+else()
+    set(CMAKE_PROJECT_VERSION_TWEAK 0)
+    set(CMAKE_PROJECT_VERSION "${OpenVINO_VERSION}.${CMAKE_PROJECT_VERSION_TWEAK}")
+endif()
+
+# adding CI build number
+if (DEFINED ENV{CI_BUILD_NUMBER})
```
Comment (ilya-lavrenov) [inline modules/custom_operations/CMakeLists.txt:27]: I supposed that `CI_BUILD_NUMBER` fully overrides OpenVINO defined version. 
It has a format like 2023.3.0-master-asd9490834

File: modules/custom_operations/user_ie_extensions/tokenizer/python/openvino_tokenizers/version.py [hunk 3]
```diff
@@ -0,0 +1 @@
+__version__ = '2023.3.0.0'
```
Comment (apaniukov) [inline modules/custom_operations/user_ie_extensions/tokenizer/python/openvino_tokenizers/version.py:1]: Maybe it is better to get a version from openvino and then add `.0` to it? That way we didn't need to manually set it every time.

**Comment by mryzhov** [comment]
not required anymore

**Pull request closed without merging**